AmplifyPlugins/API/AWSAPICategoryPlugin/Support/Utils/GraphQLRequest+Extension.swift

```diff
@@ -0,0 +1,20 @@
+//
```

Comment:
nit: filename should probably be `GraphQLRequest+toOperationRequest` or something similar